addons/mail/static/src/discuss/call/common/peer_to_peer.js

```diff
@@ -663,6 +671,11 @@ export class PeerToPeer extends EventTarget {
         }
         this._isPendingNotify = true;
         await new Promise((resolve) => setTimeout(resolve, this._batchDelay));
+        if (!this.isActive) {
+            this._isPendingNotify = false;
+            this._notificationsToSend.clear();
```

Comment:
If reset is done in disconnect, do we still need it here?

We should just make sure there is no flux were we set _isPendingNotify true or push in the list when it is disconnected.